docs/admin/configuration/authentication-and-user-management/keycloak-user.md

```diff
@@ -1,83 +1,94 @@
 ---
 sidebar_position: 4
 id: keycloak-user
-title: adding user with keycloak
+title: Adding user with Keycloak
 description: Adding user with Keycloak
 draft: true
 ---
 
 # Creating New Users in Keycloak for OpenCloud
 
-This guide explains how to create new users in Keycloak for OpenCloud, including guest users without personal spaces. While OpenCloud currently does not have a built-in "invite external user" feature, this functionality can be replicated using Keycloak.
+This guide explains how to create new users in Keycloak for OpenCloud, including "User Light" without personal spaces. While OpenCloud currently does not have a built-in "invite external user" feature, this functionality can be replicated using Keycloak.
 
 ## Background
 
 One of the most frequently requested features by administrators has been support for guest or external users. Previously, this was discussed as adding external users during the sharing process. These users were provisioned on the fly and received an invite link.
 
 Although OpenCloud does not natively support this method, similar functionality can be achieved using Keycloak for user management.
 
-## Step 1: Assign Admin Permissions in Keycloak
+## Assign Admin Permissions in Keycloak
 
 To manage users and groups for OpenCloud, you need a user with administrative privileges in the Keycloak realm.
 
-1. Log in to Keycloak as an admin.
-2. Navigate to the OpenCloud realm:  
-   [https://keycloak.keycloak-daily.opencloud.rocks/admin/openCloud/console/#/openCloud](https://keycloak.keycloak-daily.opencloud.rocks/admin/openCloud/console/#/openCloud)
-3. Assign appropriate roles (such as `realm-admin`) to the user you want to promote.
+- Log in to Keycloak as an admin.
+- Navigate to the OpenCloud realm
+- Assign appropriate roles (such as `manage-user` and `view-users`) to the user you want to promote.
 
-> Example: A user named `dennis` is assigned as a Realm Admin.
+Example: A user named `dennis` is assigned as a Realm Admin.
+
+<img src={require(".././img/keycloak/add-user/set-admin-roles.png").default} alt="Add admin roles to user" width="1920"/>
 
 Once assigned, the user can log in as a Realm Administrator and access user and group management.
 
-## Step 2: Add New Users or Groups
+## Add New User with standard rights (no Space)
 
 With admin permissions, you can now create users and groups:
 
-- Navigate to the Users section in the Keycloak Admin Console.
-- Click Add User.
-- Fill in the required user details (e.g., username, email).
-- Optionally assign the user to one or more groups.
-- Assign roles to define their access level.
+- Login in Keycloak OpenCloud Realm with the user who has admin rights under `https://keycloak.YOUR.DOMAIN/admin/openCloud/console/#/openCloud`
+
+- Navigate to the "Users" section in the Keycloak Admin Console
 
-### Recommended Role: `OpenCloudGuest`
+- Click Add User
```

Comment:
I would add a NOTE: Our keycloak example uses `OpenCloudGuest` as the default role when registering new users. This configuration can be done in the realm settings of the openCloud realm
<img width="1488" height="936" alt="ScreenShot Tool -20250804204439" src="https://github.com/user-attachments/assets/48c9f9b0-f9ba-41df-9b13-6e2a44da52f3" />